installer/vault_key.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import pprint
+
+from onepassword import OnePassword
+
+
+class VaultKeyRetriever:
+    def __init__(self):
+        self.op = OnePassword()
+        vault_keys_doc = self.op.get_item(uuid="dg5afgiadsffeklfr6jykqymeu")
```

Comment:
This is...just a magic constant that points to an already-extant document?  Should it be a parameter?  I feel like it should be settable (maybe as an init arg?) so that Dominique et al, as they develop their own vaults, don't have to modify the code?  Or maybe I don't understand what's going on here.

Reply:
Sounds good, I forgot about that!  Will do that.